client/src/routes/app.ts

```diff
@@ -0,0 +1,39 @@
+import Login from '../pages/Login/Login';
+import Signup from '../pages/SignUp/SignUp';
+import Dashboard from '../pages/Dashboard/Dashboard';
+import Settings from '../pages/Settings/Settings';
+import { AccountType } from '../types/AccountType';
+
+export interface appRouteType {
```

Comment:
Generally interfaces should be PascalCase so -> `AppRouteType`

Reply:
noted and updated - thanks!!